doc/microsite/src/main/tut/sbt/00_sbt.md

```diff
@@ -263,6 +263,18 @@ lazy val myProjectBom = inRoot.as.module
 
 ```
 
+Imagine, you have a sdk project and you want to inclide one of it's library artifacts (i.e. target_libraty).
```

Comment:
Make it flow with previous content in the section, drop "imagine" – describe and include code samples for generated output of BOM plugin.